README.rst

```diff
@@ -251,9 +251,21 @@ Ubuntu and derivatives
 
 - Elementary OS 0.2 (based on Ubuntu 12.04)
 - Linaro 12.04
-- Linux Mint 13/14/16/17/18
+- Linux Mint 13/17/18
 - Trisquel GNU/Linux 6 (based on Ubuntu 12.04)
-- Ubuntu 10.x/11.x/12.x/13.x/14.x/15.x/16.04
+- Ubuntu 12.04/14.04/16.04
+
+Ubuntu Best Effort Support: Non-LTS releases 
```

Comment:
Okay, this looks like another sub-section to me, so it can be marked accordingly:
```
Ubuntu Best Effort Support: Non-LTS releases
********************************************
```

Maybe we could add some table of contents later.